src/addons/sidebar/ListItem.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import type { AddOnListItem } from "../browser/AddOnListItem.svelte";
+  import Pin from "../../common/Pin.svelte";
+
+  export let addon: AddOnListItem;
+
+  $: url = `#add-ons/${addon.repository}`;
+</script>
+
+<style>
+  .addon {
+    display: flex;
+    align-items: flex-start;
+    justify-content: flex-start;
+  }
+
+  h4 {
+    margin: 0em auto 1em 0.5em;
+  }
+</style>
+
+<div class="addon">
+  <a href={url}><Pin active={addon.active} /></a>
```

Comment:
Do we want to allow users to unpin directly from the sidebar? If not, we should use the Pin icon (`../../common/icons/Pin.svelte`) instead of the interactive Pin component (`../../common/Pin.svelte`).

That being said, it seems handy to be able to unpin items from the sidebar?

Reply:
I'm back and forth on whether we want to unpin from the sidebar. If you unpin, it'll disappear and it's sort of hard to get back. You have to open the browser, find it again and re-pin. Right now, it's more work to unpin, and I think that's the way I lean. So let's use the icon component instead of the interactive component.